README.md

```diff
@@ -1,2 +1,221 @@
+
+<!--
+####################
+# TABLE OF CONTENTS
+####################
+-->
+# Table of contents
+
+* [Introduction](#introduction)
+* [Usage](#usage)
+    * [For players](#for-players)
+        * [Capturing the other clan's treasure](#capturing-the-other-clans-treasure)
+        * [Preventing the other clan from capturing your treasure](#preventing-the-other-clan-from-capturing-your-treasure)
+        * [Getting kills](#getting-kills)
+    * [For clan leaders](#for-clan-leaders)
+        * [Managing your clan members](#managing-your-clan-members)
+        * [Managing your clan treasure](#managing-your-clan-treasure)
+    * [For admins](#for-admins)
+        * [Joining and leaving a clan](#joining-and-leaving-a-clan)
+        * [Promoting and demoting clan leaders](#promoting-and-demoting-clan-leaders)
+        * [Changing chat colours](#changing-chat-colours)
+        * [Creating smart signs](#creating-smart-signs)
+* [Contribution](#contribution)
+
+<!--
+###############
+# INTRODUCTION
+###############
+-->
+# Introduction
+
+Milk Wars is mostly a Capture The Flag (CTF) like event between two clans, the Milk Drinkers and the Wool Wearers. Internally, and in commands, these are referred to as `cows` and `sheep`. Clan members are able to steal the opposite clan's treasure from their clan hall and bring it back to theirs for points. You can only do this once per day to prevent farming. But good luck finding the clan halls! They're hidden and may even be booby-trapped. The only way to identify a clan hall's location is the *clan tower* on it that's identical for both clans.
+
+The other aspect of Milk Wars is the PVP. Players can kill opposite clan members *Only if a chooses, by being signed in to their clan.* If they are signed out PVP is prohibited, so those not wishing to participate in the PVP can opt out of it. If you choose to participate however, check over your shoulders once in a while because you could get killed anywhere, anytime!
+
+At the end of the event, the clan with the most treasure captures and kills will win. Signs at spawn will automatically update with the latest tally of kills and captures for each clan, so make sure to help your clan win!
+
+<!--
+########
+# USAGE
+########
+-->
+# Usage
+
+How you use this plugin depends on who you are:
+1. a regular player
+2. a leader of one of the clans
+3. an admin of the server.
+
+
+## For players
+
+Thanks for choosing to play and participate in the official Geek NZ Milk Wars! There's a couple of things you should know before you go into it, including how to score points for your clan and geek etiquette.
+
+### Capturing the other clan's treasure
+
+To capture the enemy's treasure, firstly you must be signed in to your clan. You can do this at your clan's hall. At their hall there may be traps or parkour to do in order to obtain their treasure, so be careful! Right click it to pick it up, and return it to your clan's treasure to score a point.
```

Comment:
Might want to mention that you can't hold certain items when right clicking the treasure
```suggestion
To capture the enemy's treasure, firstly you must be signed in to your clan. You can do this at your clan's hall. At their hall there may be traps or parkour to do in order to obtain their treasure, so be careful! Right click it with an empty hand to pick it up, and return it to your clan's treasure to score a point.
```